packages/amplify-provider-awscloudformation/lib/configuration-manager.js

```diff
@@ -485,19 +490,24 @@ function removeProjectConfig(context) {
   }
 }
 
-async function loadConfiguration(context, awsClient) {
-  const projectConfigInfo = getCurrentConfig(context);
+async function loadConfiguration(context) {
+  const { envName } = context.amplify.getEnvInfo();
+  const config = await loadConfigurationForEnv(context, envName);
+  return config;
+}
+
+async function loadConfigurationForEnv(context, env) {
```

Comment:
why adding the env argument if it is not used at all, env is read from the config info file directly.

Reply:
This was caused when rebasing from master